pkg/metrics/collector.go

```diff
@@ -0,0 +1,126 @@
+package metrics
+
+import (
+	"context"
+	"github.com/go-logr/logr"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	k8smetrics "sigs.k8s.io/controller-runtime/pkg/metrics"
+
+	"github.com/aquasecurity/trivy-operator/pkg/apis/aquasecurity/v1alpha1"
+	"github.com/aquasecurity/trivy-operator/pkg/operator/etc"
+)
+
+var (
+	imageVulnLabels = []string{
+		"report_namespace",
+		"report_name",
```

Comment:
I am not sure how to name the namespace and name labels. The term `report_name` is borrowed from starboard-exporter, but I am not sure if it is correct in this context. Any input/comments?

Reply:
Well, the name of the metric already references the object. So I'd say that simply `name` and `namespace` are fine.